app/components/component-tree-item.hbs

```diff
@@ -44,9 +44,7 @@
           {{#each @item.args.positional as |value|}}
             <div class="arg-token flex ml-2">
               {{if (is-string value) "\""}}
```

Comment:
We can save this refactor for another PR since this looks almost ready to go, but it had occurred to me that we should probably move the surrounding quotes and mustache (optional, via a `@mustache={{true}}` argument) into the component template, and turn `isString` into a getter instead of a global helper. Just wanted to note this somewhere so we don't forget.

Reply:
Sounds good @chancancode